pkg/kando/stream_push.go

```diff
@@ -95,3 +97,23 @@ func GenerateStreamPushCommand(configFile, dirPath, filePath, password, sourceEn
 		strings.Join(kandoCmd, " "),
 	}
 }
+
+// enclosePassword adds escape char \ before every double quote " or backtick ` in the password
+// and then encloses the result between " (double quote) to handle special chars in the password
+func enclosePassword(password string) string {
+	// add escape char for "
+	reg, err := regexp.Compile("\"")
+	if err != nil || len(password) == 0 {
+		return password
+	}
```

Comment:
```suggestion
	if password == "" {
		return password
	}

	// add escape char for "
	reg, err := regexp.Compile("\"")
	if err != nil {
		return password
	}
```
Return early to avoid regexp compile.